tools/terser.d.ts

```diff
@@ -192,12 +192,13 @@ export interface MinifyOptions {
 
 export interface MinifyOutput {
     code?: string;
-    map?: RawSourceMap | string;
+    map?: EncodedSourceMap | string;
+    decoded_map?: DecodedSourceMap | null;
 }
 
 export interface SourceMapOptions {
     /** Source map object, 'inline' or source map file content */
-    content?: RawSourceMap | string;
+    content?: SectionedSourceMapInput | string;
```

Comment:
```suggestion
    content?: SectionedSourceMapInput | EncodedSourceMap | string;
```

RawSourceMal is different from SectionedSourceMapInput, I've noticed. Can trace-mapping accept its equivalent?

Reply:
It's [already included](https://github.com/jridgewell/trace-mapping/blob/10c574ec8e288a862ae6b00a0878bcfb6665f2a0/src/types.ts#L58-L59):

```typescript
type SectionedSourceMapInput = SourceMapInput | SectionedSourceMap;
```

With `SectionedSourceMap` being the maps with a `sections` field. And `SourceMapInput` being:

```typescript
type SourceMapInput = string | EncodedSourceMap | DecodedSourceMap | TraceMap;
```

Reply:
I see 😅 sorry